tests/integration/test_gitlab.py

```diff
@@ -97,6 +97,23 @@ def test_parent_project(self):
         assert self.project.get_fork().parent.namespace == "packit-service"
         assert self.project.get_fork().parent.repo == "ogr-tests"
 
+    def test_commit_statuses(self):
+        flags = self.project.get_commit_statuses(
+            commit="24c86d0704694f686329b2ea636c5b7522cfdc40"
+        )
+        assert isinstance(flags, list)
+        assert len(flags) == 0
```

Comment:
It would be nice to have also some tests where we have some flags.

You can rename this test to something like `test_commit_statuses_empty` and add one with some flags...